### PR TITLE
WIP: converted list & confirm to promt toolkit 2

### LINKED ISSUE #24: Allow choice of Pointer
For the list/checkbox types, the pointer is the unicode character U+276F. Windows command prompt seems to have an issue with displaying that character (see screenshot). Note: Notepad displays the symbol correctly using the same font, so it's not an issue with the font. Would be nice if we could choose the pointer symbol too, just like we can choose the 'qmark' symbol. Thanks.

capture2

### PR DESCRIPTION
**Pull request by tmbo**
Goal:
Support promt toolkit 2

Description:
- confirm works with promt toolkit 2
- list works with promt toolkit 2

to try it out:
```bash
pip install -U promt_toolkit
python examples/confirm.py
python examples/list.py
```

promt TODO:
- support:
```
patch_stdout=patch_stdout,
return_asyncio_coroutine=return_asyncio_coroutine,
true_color=true_color,
refresh_interval=refresh_interval,
eventloop=eventloop)
```

Related to #14 
Fixes #24

### CODE REVIEW

**Comment by CITGuru** [comment]
Nice one @tmbo

File: PyInquirer/prompts/list.py [hunk 1]
```diff
@@ -67,118 +63,121 @@ def _init_choices(self, choices, default=None):
...
 
-    # key bindings
-    manager = KeyBindingManager.for_prompt()
+    bindings = KeyBindings()
 
-    @manager.registry.add_binding(Keys.ControlQ, eager=True)
-    @manager.registry.add_binding(Keys.ControlC, eager=True)
+    @bindings.add(Keys.ControlQ, eager=True)
+    @bindings.add(Keys.ControlC, eager=True)
     def _(event):
```
Comment (CITGuru) [inline PyInquirer/prompts/list.py]: There's a bug here. It throwing a KeyboardInterrupt() event which is being cacth and this makes the current loop hard to exit. You have to answer all the questions.

Comment (tmbo) [reply PyInquirer/prompts/list.py]: I've fixed the keyboard interrupt - it should now be handled they way it was before.

File: PyInquirer/utils.py [hunk 2]
```diff
@@ -29,7 +31,46 @@ def colorize_json(data):
...
+    args = arguments_of(func)
+
+    if defaults:
+        args = args[:-len(defaults)]
+    return args   # all args without default values
+
+
+def missing_arguments(func, argdict):
+    """Return all arguments that are missing to call func."""
+    return set(required_arguments(func)) - argdict.keys()
```
[the remark this answers is not in the document]
Comment (tmbo) [reply PyInquirer/utils.py]: :+1:

**Pull request closed without merging**